test/specs/absolute-root/absolute-root.spec.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+const { expect } = require("chai");
+const $RefParser = require("../../..");
+const path = require("../../utils/path");
+const url = require("../../../lib/util/url");
+
+describe("When executed in the context of root directory", () => {
+  const { cwd } = url;
+  const { cwd: processCwd } = process;
+
+  beforeEach(() => {
+    url.cwd = function () {
+      try {
+        process.cwd = () => "/";
+        return cwd.apply(null, arguments);
+      }
+      finally {
+        process.cwd = processCwd;
+      }
+    };
+  });
+
+  afterEach(() => {
+    url.cwd = cwd;
+    process.cwd = processCwd; // already restored at line 19, but just in case
+  });
+
+  it("should resolve successfully a $ref pointing at absolute location", async () => {
```

Comment:
To match the other test suites, there need to be tests for the `parse()`, `resolve()`, `dereference()`, and `bundle()` methods, each with absolute paths, relative paths, and URLs.  I recommend copying the contents of "test/specs/external" into this directory and using that as your starting point

Reply:
> relative paths

This is not going to work since I spoof `process.cwd`. I'd need to spoof fs calls as well.
Are you okay with that?